### DISCUSSION TITLE
Collecting awesome docs from established communities to generalize and upstream for other communities

### DISCUSSION THREAD

**Discussion by jnywong** [2i2c-org/docs#237] [Ideas]
It's _**so**_ incredible to see contextualized documentation of established 2i2c communities have written and self-organized around their hubs, and I was wondering if we can use the power of open source to upstream generalized documentation to the [2i2c Service Guide](https://docs.2i2c.org/) to help other communities who are starting out 🧙‍♀️

[WIP] Here is a list for me to remember the awesome technical content community champions have created

- https://github.com/leap-stc/leap-stc.github.io/pull/130
- https://book.cryointhecloud.com/intro.html
- https://nasa-openscapes.github.io/earthdata-cloud-cookbook/how-tos/using-s3-storage.html

Feel free to highlight any other examples in the discussion below!

**Attribution**

All of the hard work that goes into developing technical content deserves to be attributed. Source material will be acknowledged accordingly ❤️ 

---

Discussion inspired by @jbusecke 

> Pinging @jnywong here. I think that the guide we are expanding on here could also be relevant for upstream docs. Wondering if there is a 'collection issue' where I could add links to stuff I think would be helpful for other communities (and their champions in particular).

_See comment https://github.com/leap-stc/leap-stc.github.io/pull/130#issuecomment-2186697984_

**Comment by choldgraf** [comment]
I think this is a fantastic idea - also excellent for showing off value in 2i2c's service. When you share something like the cryocloud docs it is much easier for somebody to "get it" compared w describing the service in the abstract.

https://book.cryointhecloud.com/intro.html

If / when we facilitate more MyST usage, then folks should be able to re-use content too...

**Reply by jbusecke** [reply]
@choldgraf are there things that are still blocked upstream (we are waiting for jupyter book 2?) or is this an actionable item on our end to refactor our docs?

**Reply by choldgraf** [reply]
I think that's a question for @jnywong - Jenny do you think there's progress that can be made with others' docs as-is or would this require action from them?

**Reply by jnywong** [reply]
> are there things that are still blocked upstream (we are waiting for jupyter book 2?)

We will need time to upgrade the [2i2c Service Guide](https://docs.2i2c.org/) to JB2 that will allow you to [embed](https://mystmd.org/guide/embed) content from 2i2c upstream into the LEAP docs. This work isn't currently planned, but we might have capacity to look at this at the end of the year :)

In the meantime, you can upgrade the LEAP docs to JB2 so that the embed functionality is readily available.

**Comment by jnywong** [comment]
Yes, embrace the content re-use! We don't want the communities we serve to write the same general docs x100 about getting started with X on the hub – we want to help them focus on the science 🚀
